### DISCUSSION TITLE
شناسه پرداخت در درگاه بانک ملت

### DISCUSSION THREAD

**Discussion by omid811** [Sina-Soltani/Parbad#221] [Q&A]
درگاه بانک ملت برای پرداخت اینترنتی فیلدی برای دریافت شناسه پرداخت داره. 
از طریق پرباد چطور میشه این شناسه رو برای پرداخت ارسال کرد؟

شناسه پرداخت از طریق زیر برای درگاه بانک ملت میتونه ارسال بشه
تابع: bpPayRequest  
فیلد: payerId

**Comment by Sina-Soltani** [comment]
در نسخه فعلی موجود نیست
باید توسعه داده بشه
